consul/client.go

```diff
@@ -334,7 +334,13 @@ func (c *ConsulAlertClient) NewAlerts() []Check {
 func (c *ConsulAlertClient) CustomNotifiers() (customNotifs map[string]string) {
 	if kvPairs, _, err := c.api.KV().List("consul-alerts/config/notifiers/custom", nil); err == nil {
 		for _, kvPair := range kvPairs {
-			custNotifName := filepath.Base(kvPair.Key)
+			rp := regexp.MustCompile("/([^/]*)$")
+			match := rp.FindStringSubmatch(kvPair.Key)
+			custNotifName := match[1]
+			if custNotifName == "" {
+				continue
+			}
+			customNotifs = make(map[string]string)
```

Comment:
Each key from the custom notifiers directory will be a different notifier name.  So the loop will need to set the name of the custom notifier for each entry as it goes through.  Imagine 3 custom notifiers defined in the custom directory with the key name as the name of the notifier and the value is the command to run:
consul-alerts/config/notifiers/custom/
  - "SMS" -> SMScommand.sh
  - "script_action" -> somescript.pl
 - "alt_email" -> custom_email_script.rb

The loop needs to set the custNotifName for each to use as a key in the customNotifs map.